Release note. (#1497)
scverse/scvi-tools: docs/release_notes/v0.16.0.md

# New in 0.16.0 (2022-04-08)

> In this release, we have refactored our destVI model:

Reviewer: see a previous release note for how to reference a model (will automatically hyperlink it to the docstring). I'd use this reference notation each time you refer to a model by name

> 1. Bug fix in cell type amortization, which leads to on par performance of cell type amortization with free parameter for cell type.

Reviewer: I would add in parenthesis the parameter name and the argument so people know what arguments this corresponds to

2. Bug fix in library size, that lead to dependency between sum over cell type proportions and library size.

> 3. Vamp_prior_p is not computed anymore on random subset of single cells but cell type specific subclustering using cluster variance,

Reviewer: Use variable names and code notation in markdown (wrap with `)

cluster mean and cluster mixture proportion for computation. This leads to more stable results and faster computation time.

> Setting vamp_prior_p to the expected resolution is critical in this application.

Reviewer: same here, wrap code variables in `

4. The new default is to also use dropout during the decoder of CondSCVI and subsequently in DestVI, we found this to be beneficial after
bug fixes listed above.
5. We changed the weighting of the loss on the variances of beta and the prior of eta.

> **Important Note:** Due to bug fixes listed above both versions of destVI are not compatible. Despite instability in training in the

Reviewer: Use this notation

```
:::{note}
Due to...
:::
```

outdated version, we were able to reproduce results generated with this code. We therefore do not strictly encourage to rerun old
experiments.

We published a new tutorial. This new tutorial incorporates a new utility package [destvi_utils](https://github.com/YosefLab/destvi_utils) that generates exploratory plots of the results of destVI. We refer to the manual of this package for further documentation.

## Changes

- Major refactor of destVI ([#1457]).

## Breaking changes

## Contributors

- [@jjhong922]
- [@adamgayoso]

> - [@watiss]

Reviewer: add you and romain and me here

In this release, we have refactored our {class}`~scvi.model.DestVI`:

1. Bug fix in cell type amortization, which leads to on par performance of cell type amortization `V_encoder` with free parameter for cell type proportions `V`.
2. Bug fix in library size in {class}`~scvi.model.CondSCVI`, that lead to downstream dependency between sum over cell type proportions `v_ind` and library size
`library` in {class}`~scvi.model.DestVI`.
3. `neg_log_likelihood_prior` is not computed anymore on random subset of single cells but cell type specific subclustering using cluster variance `var_vprior`,
cluster mean `mean_vprior` and cluster mixture proportion `mp_vprior` for computation. This leads to more stable results and faster computation time.
Setting `vamp_prior_p` in {func}`~scvi.model.DestVI.from_rna_model` to the expected resolution is critical in this algorithm.
4. The new default is to also use dropout `dropout` during the decoder of {class}`~scvi.model.CondSCVI` and subsequently `dropout_decoder` in {class}`~scvi.model.DestVI`,
we found this to be beneficial after bug fixes listed above.
5. We changed the weighting of the loss on the variances of beta and the prior of eta.

:::{note}

> Due to bug fixes listed above both versions of {class}`~scvi.model.DestVI` are not compatible. Despite instability in training in the

Reviewer: ```suggestion
Due to bug fixes listed above this version of {class}`~scvi.model.DestVI` is not backwards compatible. Despite instability in training in the
```
